specs/StartInBackgroundMode.md

```diff
@@ -0,0 +1,101 @@
+CreationPriority
+===
+
+# Background
+WebView2 has a functionality enabled by default which raises the launch priority of WV2
+processes to ensure that WV2 processes are given higher CPU priority and access to resources 
+while creation, and that can bring improvements on time spent creating a WV2 instance. Since 
+this improvement is at the cost of taking higher CPU priority, we introduce `CreationPriority` 
+property to give developers an option to decide by themselves.
+
+# Description
+* Setting `CreationPriority` to `COREWEBVIEW2_CREATION_PRIORITY_NORMAL` will signal WebView2 
+to use less system resources during the creation. 

```

Comment:
Is this correct? Its the creation of the browser process that benefits from the priority?

```suggestion
to have normal priority compared to other processes during the creation of the WebView2 browser process.
```

Reply:
"use less system resources" is not accurate. Rephased!